e2e/wdio/platform/fixtures/testData/form-generator.ts

```diff
@@ -0,0 +1,4 @@
+export const simplePassword = '123';
+export const correctPassword = 'FundamentalNgx12#';
+export const incorrectBirthdayYear = '1/1/2020';
```

Comment:
please rename:
invalidBirthday
validBirthday

also, please use a method to get current year so that the invalid birthday will always be invalid and not need to be manually updated every year.